html-files/pages/Page10.html

```diff
@@ -30,72 +31,29 @@
-                        </li>
-                        <li class="u-nav-item"><a
-                                class="u-active-grey-5 u-border-active-palette-1-base u-border-hover-palette-1-base u-button-style u-hover-grey-10 u-nav-link u-text-active-grey-90 u-text-grey-90 u-text-hover-grey-90"
-                                href="/html-files/support-pages/About.html" style="padding: 10px 20px;">About</a>
-                                <li class="u-nav-item"><a class="u-button-style u-nav-link" href="/html-files/support-pages/Contact.html"
-                                    style="padding: 10px 20px;">Contact</a>
-                                </li>
-                        </li>
-                    </ul>
-                </div>
-                <div class="u-custom-menu u-nav-container-collapse">
-                    <div class="u-black u-container-style u-inner-container-layout u-opacity u-opacity-95 u-sidenav">
-                        <div class="u-inner-container-layout u-sidenav-overflow">
-                            <div class="u-menu-close"></div>
-                            <ul class="u-align-center u-nav u-popupmenu-items u-unstyled u-nav-2">
-                                <li class="u-nav-item"><a class="u-button-style u-nav-link" href="/index.html"
-                                        style="padding: 10px 20px;">Home</a>
-                                </li>
-                                <li class="u-nav-item"><a class="u-button-style u-nav-link" href="Page1.html"
-                                        style="padding: 10px 20px;">Recipies</a>
-                                </li>
-                                <li class="u-nav-item"><a class="u-button-style u-nav-link" href="/html-files/support-pages/About.html"
-                                        style="padding: 10px 20px;">About</a>
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
-                    <div class="u-black u-menu-overlay u-opacity u-opacity-70"></div>
-                </div>
-            </nav>
-        </div>
-    </header>
+    <header>
+        <nav>
+          <div class="logo">
+              <a href="index.html"><img height="50" src="https://ik.imagekit.io/daringbaba/Food_Coders/Screenshot_2022-12-23_160309-removebg-preview.png?ik-sdk-version=javascript-1.4.3&updatedAt=1671792391543" alt="Rosa Logo"></a>
+          </div>
+          <div class="toggle">
+              <span class="first"></span> 
+              <span class="middle"></span>
+              <span class="last"></span>
+          </div>
+          <div class="navigation-bar">
+              <ul>
+                  <li class="active"><a href="../../index.html">Home<span class="underline"></span></a></li>
+                  <li><a href="./html-files/pages/Page1.html">Recipes<span class="underline"></span></a></li>
```

Comment:
use ```./Page1.html``` instead. This error is causing the site to have wrong redirection to all the pages Mentioned below -
- Page 2
- Page 4
- Page 5
- Page 6
- Page 7
- Page 8
- Page 9
- Page 10
- Page 11
- Page 12
- Page 13

when the user taps on ```Recipes``` button in the Navbar